design/kanister-prometheus-integration.md

```diff
@@ -215,7 +215,7 @@ controller/metrics.go
 
 ```golang
 const (
-	ACTION_SET_COUNTER_VEC_LABEL_RES     = "resolution"
```

Comment:
[nit] I think we should exclude `./design` directory from the link checks

Reply:
This is not checking linter on `./design` package. The change ActionSetCounterVecLabelRes is made because this is the variable referred in code, and there changes needs to be done.